lib/tableau_dev_server/router.ex

```diff
@@ -32,11 +33,21 @@ defmodule TableauDevServer.Router do
   end
 
   defp recompile(conn, _) do
-    if conn.request_path != "/ws" do
-      WebDevUtils.CodeReloader.reload()
+    if conn.request_path == "/ws" do
+      conn
+    else
+      case WebDevUtils.CodeReloader.reload() do
+        {:error, errors} ->
+          errors = Enum.filter(errors, &(&1.severity == :error))
+          message = Enum.reduce(errors, "", fn error, acc -> "#{acc}\n#{error.message}" end)
+          stacktrace = List.first(errors).stacktrace
```

Comment:
this can fail if the list is is only non error severity, which would make List.first return nil

Reply:
It should not fail as we are expecting error, first value in tuple is `:error`. 
For the `:ok` it might return `Mix.Task.Compiler.Diagnostic` with warnings.